typescript/sdk/src/providers/MultiProvider.ts

```diff
@@ -43,7 +55,7 @@ export interface MultiProviderOptions {
 export class MultiProvider<MetaExt = {}> extends ChainMetadataManager<MetaExt> {
   readonly providers: ChainMap<Provider>;
   readonly providerBuilder: ProviderBuilderFn<Provider>;
-  signers: ChainMap<Signer>;
+  signers: ChainMap<ZKSyncWallet | Signer>;
```

Comment:
So ZKSyncWallet doesn't implement the Signer interface? I'd expect that it would since an Ethers.js Wallet does implement it

Reply:
For types we can remove both ZKSyncWallet and ZKSyncProvider... but we need to be mindful that MultiProvider is also used for deploying contracts (& gas estimation...), and for that we use ZKSyncDeployer class, which strictly requires that the type of signer is ZKSyncWallet [extends Wallet base]. Meaning that type casting to ZKSyncWallet will still be required down in the code. We need to keep this in mind.

Reply:
This is resolved, only Signer is used now